zkvm/README.md

```diff
@@ -2,6 +2,9 @@
 
 An evolution of TxVM with **cloaked assets** and **zero-knowledge smart contracts**.
 
+* [ZkVM specification](docs/zkvm-spec.md) — transaction validation rules.
+* [ZkVM implementation guide](docs/zkvm-impl.md) — transaction validation rules.
```

Comment:
This should be something like "how to create transactions" instead of "transaction validation rules", I think

Reply:
oops